src/operators/operators.cpp

```diff
@@ -513,36 +501,28 @@ void OperatorBase<dim,nstate,real>::get_FR_correction_parameter (
     if(c_input == FR_enum::cHU || c_input == FR_enum::cHULumped){ 
         const double pfact = compute_factorial(curr_cell_degree);
         const double pfact2 = compute_factorial(2.0 * curr_cell_degree);
```

Comment:
Consider moving each block to its own function with the a relevant reference to where those coefficients are defined, and have a switch statement call those various functions.

Reply:
I decided to leave it as it is since each block is just a few lines.

Reply:
A few lines is the perfect amount of lines for a function.

Reply:
Fair enough, fixed this and gave citations for each function value.